course_discovery/apps/course_metadata/models.py

```diff
@@ -330,7 +330,10 @@ def organization_name(self):
 
 class Course(TimeStampedModel):
     """ Course model. """
-    partner = models.ForeignKey(Partner)
+    org = models.CharField(
```

Comment:
- Removed field `partner` and added field `org` to table "course_metadata_coruse".